qbft/state.go

```diff
@@ -81,14 +77,23 @@ func (c *Config) GetTimer() Timer {
 	return c.Timer
 }
 
+func (c *Config) GetCutOffRound() int {
+	return c.CutOffRound
+}
+
+// GetSignatureVerifier returns the verifier for operator's signatures
+func (c *Config) GetSignatureVerifier() types.SignatureVerifier {
+	return c.SignatureVerifier
+}
+
 type State struct {
-	Share                           *types.Share
+	Share                           *types.Operator
```

Comment:
As I understand, `Share` represents  what we used to call `Share` for non-committee roles, and `Operator` represents it for committee roles. If it's correct, then why does `State` support `Share`? If not correct, what's the difference between `Share` and `Operator` and why do we need both?

Reply:
Pretty much the former Share type held both operator and validator share metadata.
`Operator` is a pure SSV entity and `Validator` is an eth entity.
We now decoupled those 2 domains to `Operator` and `Share`.
I did add comments on these types. Maybe they were not clear enough?

The word `Share` implies the entity is part of a committee, so both `Operator` and `Share` types are called `Share` as variables. I admit that part of the reason the `Share` variable didn't change the name is to have less conflicts and changes for this huge PR. Maybe it should be renamed `OperatorShare` or some other name? Or maybe `share` is a clear short name? If you have some thoughts for better naming do share?

Reply:
@GalRogozinski thanks for the explanation. I don't have any better ideas right now, I'll leave a comment if something comes to my mind